zoom_monitor.py

```diff
@@ -148,48 +152,34 @@ def _open_participants_list(self):
                 time.sleep(1)
                 continue  # Go to the next attempt
 
-            selected = self._is_participants_button_selected()
             try:
-                # Clicking on the participants list only selects the button.
-                # As a small clue: if a human clicks on it, the mouse-down
-                # selects the button while the mouse-up opens the participants
-                # list. Double-clicking on it seems to work okay (maybe the
-                # second click implicitly creates a mouse-up on the first one).
-                # If the button is already selected, only one click is needed.
+                # Sometimes, the button is hidden off the bottom of the window,
+                # but moving the mouse to it will make it visible again. This
+                # tends to happen after someone stops sharing their screen.
+                ActionChains(self._driver).move_to_element(button).perform()
```

Comment:
Also move this outside the try block: it should not have any exceptions, and making the try block as narrow as possible makes it easier to reason about.